[ADD] [16.0] sale_stock_expiry_date_on_qty_at_date_widget (#3594)
OCA/sale-workflow: sale_stock_expiry_date_on_qty_at_date_widget/readme/CONTEXT.md

> This module was developed because sometimes Commercial users want to know where is the Expiraiton Date of the products.

Reviewer: ```suggestion
This module was developed because sometimes salesperson users need to know when products will expire.
```

> It will be useful for you if your Commercial users need to share this informations with its customers.

Reviewer: ```suggestion
It will be useful for you if your salespersons need to share this information with their customers.
```
